source/content/shibboleth-sso.md

```diff
@@ -209,7 +209,7 @@ $conf['simplesamlphp_auth_installdir'] = $_ENV['HOME'] .'/code/private/simplesam
 
 ```php:title=settings.php
 # Provide universal absolute path to the installation.
-$settings['simplesamlphp_dir'] = $_ENV['HOME'] .'/code/private/simplesamlphp';
+$settings['simplesamlphp_dir'] = $_ENV['HOME'] .'/code/vendor/simplesamlphp/simplesamlphp';
```

Comment:
Hmmm, I'm curious because [this section](https://pr-6963--pantheon-docs.my.pantheonfrontend.website/docs/shibboleth-sso#drupal-8-settings) is talking only about non-composer sites; vendor is probably only there for composer sites. Also, if you go to "Download Method" at the beginning of the doc, it instructs you to put the file at `private/simplesamlphp` so I'm unsure about why we want to change this path

Reply:
Thanks Kevin

Reply:
The issue [states](https://github.com/pantheon-systems/documentation/issues/6949) , "  Failed opening '//code/private/simplesamlphp/lib/_autoload.php'...". The reason for this is that although the config dirs are symlinked from /code/private/simplesamlphp, other dirs (such as lib) are not."  Is this not true

It also mentions
"For composer installs (not for manual installs), the instructions should recommend that it be set to '/code/vendor/simplesamlphp/simplesamlphp'"

WDYT?

Reply:
As far as I understand, if you're using composer, you don't even need to set that variable. I checked at simplesamlphp_auth module and found that this variable is used to include an autoload file: https://git.drupalcode.org/search?search=simplesamlphp_dir&group_id=2&project_id=8933&scope=&search_code=true&snippets=false&repository_ref=8.x-3.x&nav_source=navbar

If using composer, the autoload from composer should take care of that and load the right files for you so that line should not be needed.

And actually, I think that was the intention of the original guide: do this only if not composer, do nothing if composer but maybe it wasn't super clear.